packages/client/package.json

```diff
@@ -15,9 +15,18 @@
   },
   "dependencies": {
     "@apollo/client": "^3.7.9",
+    "@emotion/react": "^11.10.6",
+    "@emotion/styled": "^11.10.6",
+    "@mui/icons-material": "^5.11.9",
```

Comment:
I don't think we are using mui icons

Reply:
mui icons that we used are Add and Delete